transport-protocol/src/custom.rs

```diff
@@ -170,8 +170,9 @@ pub fn serialize_quote(quote: &sgx_types::sgx_quote_t) -> transport_protocol::Sg
 }
 
 /// Serialize a program binary.
-pub fn serialize_program(program_buffer: &[u8]) -> TransportProtocolResult {
+pub fn serialize_program(program_buffer: &[u8], file_name: &str) -> TransportProtocolResult {
```

Comment:
Isn't `file_name` better passed as a `String` rather than a `&str`?  The only time it is used it is immediately converted to a `String` anyway.